### PR TITLE
Updated releaser configuration

### PR DESCRIPTION
**Pull request by jirihnidek**
* Added releaser configuration for rhel-8.9
* Updated Fedora section to support currently supported version
  of Fedora

### CODE REVIEW

File: rel-eng/releasers.conf [hunk 1]
```diff
@@ -25,3 +25,11 @@ required_bz_flags = release+
 # Change this if you wish to use a placeholder "rebase" bug if none
 # are found in the changelog.
 placeholder_bz = 
+
+[rhel-8.9]
+releaser = tito.release.DistGitReleaser
+branches = rhel-8.9.0
```
Comment (ptoscano) [inline rel-eng/releasers.conf]: 8.9+ follow the "stream" model already in use in el9, so you need to add a c8s releaser; see the equivalent for sub-man i added recently: https://github.com/candlepin/subscription-manager/commit/9a55e5348f5dcc2377857964f26896c9574f8037

**Review by ptoscano — CHANGES_REQUESTED** [review]

File: rel-eng/releasers.conf (hunk 1 above)
Comment (jirihnidek) [reply rel-eng/releasers.conf]: Oh, I forgot about this. Good point.

**Comment by wottop** [comment]
ack

**Review by ptoscano — APPROVED** [review]
thanks!

**Pull request merged**